### ISSUE TITLE
test_camera_mobilefacenet_openvino.py

### ISSUE THREAD

**Issue by baishalichaudhury** [cunjian/pytorch_face_landmark#22]
In this script the face detector is the RBF network, thats pretty clear. But is the face landmark algorithm mobilefacenet, I thought mobilefacenet is face recognition algorithm. Can you please clarify or provide the reference paper for this version of mobilefacenet on which the model is based.  Thanks

**Comment by cunjian** [comment]
yes, mobilefacenet was originally proposed for face recognition. However, it is easy to modify the backbone network to fit the regression task.
